internal/tweets/validation.go

```diff
@@ -20,16 +17,16 @@ func (v *TweetValid) Error() string {
 	return result
 }
 
-func CheckTweetText(fl validator.FieldLevel, v *TweetValid) bool {
+func (s *Service) CheckTweetText(fl validator.FieldLevel, v *TweetValid) bool {
 	text := fl.Field().String()
-	if len(text) > maxLenghtTweet {
-		v.ValidErr["name"] += "long text,"
+	if len(text) > s.TweetLength {
```

Comment:
where this `s.TweetLength` is being set? and updated on change?

Reply:
ReadSettings

Reply:
this? https://github.com/mo-pm888/twitter/pull/44/files#diff-e5da3bf0e3b8f7e147d97b249c480b6abe8f681f067dad7b2b278b8d107c1ce0R50 
it runs only on start and fills in the admins service.
Here you are checking a field of tweets.Service which is not set nor updated ever